### DISCUSSION TITLE
Welcome to FoodIndustry Discussions!

### DISCUSSION THREAD

**Discussion by Oceanel51** [Oceanel51/Food-Industry#48] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BlackAngelKitty** [comment]
Hello

I've been using this mod for months now and I absolutely love it. I know it hasn't been worked on in a while and is technically deprecated, but I would like to ask for permission to work on it.

Reason is, it's a really fun mod but has a few bugs. I've already fixed a few such as possible incompatibility (The Ruins Mod/Bio Industries/Space Exploration) and issues feeding calfs that would cause the game to crash.

**Reply by Oceanel51** [reply]
Hey, how are you doing? Do you want to continue developing the mod? I'm thinking about this... recent months. 🤔

**Reply by BlackAngelKitty** [reply]
Hi! I'm well, thank you.

Yes, I fixed a few of the bugs in Factorio 1.1 and I'm currently working on compatibility for 2.0 (I'm not lying that took 18 hours just to get it to not crash, 2.0 is sooo much stricter with how mods are built, so lots of changes.)

I am keeping the documentation up to date as well for if and when it's ever deployed. I've also looked at the old bug reports, even found your Trello board with to-do's.

It really is my favourite mod and I love working on it, I even started learning how to make sprites and spritesheets in Blender.

**Comment by jonkeda** [comment]
Hi,

I think you can go ahead. Have fun with the cows.

Love to hear from you what you added.

Regards,
Daan
________________________________

**Comment by BlackAngelKitty** [comment]
Hello!

Just wanted to say I've started working on the mod again. I stopped after 2.0 was announced and didn't play for a while.

After about 18 hours (I am not kidding when I say I was losing my mind) I finally got it to load in a 2.0 game! Some things broke (new Factorio is a lot stricter, lots to change) but I'm actively getting it fully running and functional.

After that, I plan on addressing the reported bugs then continuing with the to-do list on Trello that was shared.

Question though if anyone reads this... should I push my contributions to this repo or keep it on my own and register Food Industry 3.0 as a new mod?
